package.yaml

```diff
@@ -276,12 +276,11 @@ executables:
   stack:
     main: Main.hs
     source-dirs: src/main
+    generated-other-modules: Build_stack
```

Comment:
This requires `hpack >= 0.23.0`, older versions o `hpack` will just ignore the field with a warning.  So I guess no big issue.

In addition, this will result in `cabal-version: >= 2`, which I believe is already required by `Setup.hs` (see my other PR).